website_and_docs/content/documentation/grid/getting_started.zh-cn.md

```diff
@@ -214,113 +203,83 @@ driver.quit();
-powerful (CPU, RAM) those machines are.
+选择网格角色取决于需要支持什么操作系统和浏览器、需要执行多少个并行会话、可用机器的数量以及这些机器的配置（CPU、RAM）。
 
-Creating sessions concurrently relies on the available processors to the **Distributor**. 
-For example, if a machine has 4 CPUs, the **Distributor** will only be able to create up
-to 4 sessions concurrently.
+并发创建会话依赖于 **分配器** 的可用处理器。 例如，如果一台机器有 4 个 CPU，则 **分配器** 最多只能同时创建 4 个会话。
 
-By default, the maximum amount of concurrent sessions a **Node** supports is limited by
-the number of CPUs available. For example, if the **Node** machine has 8CPUs, it can run
-up to 8 concurrent browser sessions (with the exception of Safari, which is always one).
-Additionally, it is expected that each browser session should use around 1GB RAM. 
+默认情况下，**Node** 支持的最大并发会话数受可用 CPU 数量的限制。 例如，如果 **Node** 机器有 8 个 CPU，它最多可以运行 8 个并发浏览器会话（Safari 除外，它始终是一个）。 此外，预计每个浏览器会话应使用大约 1GB 的 RAM。
 
-In general, it is a recommended to have **Nodes** as small as possible. Instead of having
-a machine with 32CPUs and 32GB RAM to run 32 concurrent browser sessions, it is better to
-have 32 small **Nodes** in order to better isolate processes. With this, if a **Node**
-fails, it will do it in an isolated way. Docker is a good tool to achieve this approach.
+通常，建议 **Nodes** 尽可能小。 与其让机器有 32 个 CPU 和 32GB RAM 来运行 32 个并发浏览器会话，不如有 32 个小的 **Node**，以便更好地隔离进程。 有了这个，如果一个 **Node** 发生故障，它将以孤立的方式进行。 Docker 是实现这种方法的好工具。
 
-Note that the default values (1CPU/1GB RAM per browser) are a recommendation and they could
-not apply to your context. It is recommended to use them as a reference, but measuring 
-performance continuously will help to determine the ideal values for your environment.
+请注意，默认值（每个浏览器 1 个 CPU/1GB RAM）是建议值，它们不适用于您的上下文。 建议将它们用作参考，但持续测量性能将有助于确定您的环境的理想值。
 
-Grid sizes are relative to the amount of supported concurrent sessions and amount of 
-**Nodes**, and there is no "one size fits all". Sizes mentioned below are rough estimations
-thay can vary between different environments. For example a **Hub/Node** with 120 **Nodes**
-might work well when the **Hub** has enough resources. Values below are not set on stone,
-and feedback is welcomed!
+网格大小与支持的并发会话数量和 **Node** 数量有关，没有“一刀切”的说法。 下面提到的尺寸是粗略的估计，不同环境之间可能会有所不同。 例如，当 **Hub** 具有足够的资源时，具有 120 个 **Nodes** 的 **Hub/Node** 可能运行良好。 以下值并非一成不变，欢迎提供反馈！
 
 ### Small
 
-**Standalone** or **Hub/Node** with 5 or less **Nodes**.
+**Standalone** 或 **Hub/Node** 不超过5个 **Nodes**.
 
 ### Middle
 
-**Hub/Node** between 6 and 60 **Nodes**.
+**Hub/Node** 介于6到60个 **Nodes** 之间。
 
 ### Large
 
-**Hub/Node** between 60 and 100 **Nodes**. **Distributed** with over 100 **Nodes**.
+**Hub/Node** 介于60到100个 **Nodes** 之间， **Distributed** 超过100个 **Nodes**。
 
 ## Warning
 
-Selenium Grid must be protected from external access using appropriate
-firewall permissions.
+必须使用适当的方法保护 Selenium Grid 免受外部访问
+防火墙权限。
 
-Failure to protect your Grid could result in one or more of the following occurring:
+未能保护您的网格可能会导致发生以下一种或多种情况：
```

Comment:
How about to keep "Grid" just like what you did in almost above parts?

Reply:
In response to the points you mentioned, I have made some changes and psuh to the patch-1 branch.